libs/@local/hash-subgraph/tests/compatibility.test.ts

```diff
@@ -89,14 +88,9 @@ test("Graph API subgraph type is compatible with library type", () => {
   // We just want to check for errors in the type when building the object, no need to use the return value
   const _subgraph: Subgraph = {
     roots: mapRoots(subgraphGraphApi.roots),
-    vertices: mapVertices(subgraphGraphApi.vertices),
+    vertices: subgraphGraphApi.vertices as Subgraph["vertices"],
```

Comment:
I think I might have given a confusing answer yesterday. Can we not delete the mappings here. It's fine to cast outside of the test files, but that's _because_ these test files are supposed to check that the shapes are the same, aside from casting for branding.

So we still need the code that expands the vertices and temporalAxes all the way out, and does a 1-1 mapping of fields, and only brands the values like `as Timestamp`

Reply:
Discussed offline and reintroduced them in:
- [Reintroduce mapQueryTemporalAxes](https://github.com/hashintel/hash/pull/1854/commits/c6712af68d4da49721f27bb1d8afa7398c3f75c7)
- [Reintroduce mapVertices](https://github.com/hashintel/hash/pull/1854/commits/e4a031935986ab191f4bf8758b8d3e349c3a17fc)

Reply:
Thank you, @Alfred-Mountfield!